package.json

```
@@ -5,15 +5,24 @@
+    "lint": "eslint . --ext .ts,.tsx",
     "lint:fix": "eslint . --fix --ext .ts,.tsx",
+    "docker:lint": "docker compose -f docker-compose.test.yml run --rm node-ci sh -lc 'npm ci && npm run lint'",
     "analyze": "cross-env ANALYZE=true npm run build",
     "type-check": "tsc --noEmit",
     "type-check:watch": "tsc --noEmit --watch",
     "test": "npm run test:node && npm run test:dom",
+    "docker:test:up": "docker compose -f docker-compose.test.yml up -d --wait postgres-test redis-test",
+    "docker:test:run": "docker compose -f docker-compose.test.yml run --rm node-ci sh -lc 'rm -rf /app/node_modules/* /app/node_modules/.prisma/* /app/.next/* 2>/dev/null || true; export DATABASE_URL=\"postgresql://postgres:${POSTGRES_PASSWORD}@postgres-test:5432/techtrend_test\" E2E_SEED_FORCE=1 LOG_LEVEL=error; npm ci --silent && npx prisma migrate deploy && npx tsx prisma/seed-test.ts && npm run test:node -- --reporters=summary --maxWorkers=2 --silent && npm run test:dom -- --reporters=summary --maxWorkers=2 --silent'",
+    "docker:test:down": "docker compose -f docker-compose.test.yml down",
+    "docker:test": "npm run docker:test:up && npm run docker:test:run; npm run docker:test:down",
+    "db:backup": "bash scripts/db/backup-dev.sh",
```

Biome (2.1.2)

[error] The key db:backup was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)